lib/bugsnag/delayed_job.rb

```diff
@@ -33,7 +33,8 @@ def error(job, error)
               p[:id]           = payload.id           if payload.respond_to?(:id)
               p[:display_name] = payload.display_name if payload.respond_to?(:display_name)
               p[:method_name]  = payload.method_name  if payload.respond_to?(:method_name)
-              p[:args]         = payload.args         if payload.respond_to?(:args)
+              p[:args]         = payload.respond_to?(:args) ? payload.args : payload.to_h
```

Comment:
Where does `to_h` come from? I didn't see this mentioned in DelayedJob anywhere.